isites_migration/views.py

```diff
@@ -29,7 +29,11 @@ def index(request):
         )
         return redirect('isites_migration:index')
 
-    processes = Process.objects.filter(name='isites_migration.jobs.migrate_files').order_by('-date_created')
+    processes = Process.objects.filter(
+        name='isites_migration.jobs.migrate_files',
+        details__at_canvas_course_id=canvas_course_id
```

Comment:
I can't find where the `at_canvas_course_id` field is set in the `details` blob in our isites_migration app, can you point it out to me?